setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+setup(
+    name='flask-sparkle',
+    version='0.1',
+    url='http://github.com/closeio/flask-sparkle',
+    license='MIT',
+    description='Close.io internal flask helpers',
```

Comment:
this isn't really an internal-only thing. It's an app for integrating the backend feed for https://sparkle-project.org/ into Flask